keps/prod-readiness/sig-storage/3301.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3301
+alpha:
+  approver: "@xing-yang"
```

Comment:
I'm not an approver for PRR.  Please pick someone from `prod-readiness-approvers`:
https://github.com/kubernetes/enhancements/blob/master/OWNERS_ALIASES#L196